packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -47,11 +47,11 @@ void _focusAndEnsureVisible(
 // sorting their contents.
 class _FocusTraversalGroupInfo {
   _FocusTraversalGroupInfo(
-    _FocusTraversalGroupScope? marker, {
+    _FocusTraversalGroupNode? group, {
```

Comment:
It could, but it's complicated by specifying the default, which could override the policy if the group is null. Maybe I'll see if I can combine that with an accessor instead.

Reply:
I looked at this, but it doesn't really save much, since I'd have to convert the saved policy member to save the given default policy instead, which doesn't really save anything, and makes the access of the policy less efficient.